packages/react-aria-components/stories/DatePicker.stories.tsx

```diff
@@ -92,11 +92,11 @@ export const DateRangePickerExample = () => (
     <Label style={{display: 'block'}}>Date</Label>
     <Group style={{display: 'inline-flex'}}>
       <div className={styles.field}>
-        <DateInput data-testid="date-range-picker-date-input" slot="start" style={{display: 'inline-flex'}}>
+        <DateInput data-testid="date-range-picker-date-input" slot="start" style={{display: 'inline'}}>
```

Comment:
sort of a breaking change
what will happen to people who have used this as an example to build their DatePicker's in the past?

Also, why is the style change different from DateField.mdx?

Reply:
so i experimented a bit and found that for the rtl segments to display in the correct order, DateInput needs to have display inline, inline-block, or block. hence why one is display inline and the other is display block. it would probably be a good idea to make it consistent and stick with display inline though. 

and as for the sort of breaking change, i think folks can continue to use `display: inline-flex`. i don't believe the changes here will affect that. the only thing that they'll run into is that if they do choose to render their datepicker in a rtl locale, it will be in the incorrect order.

Reply:
ok, so long as inline-flex doesn't break anything, I'm fine with this
we should add a little blurb to the Accessibility section for the docs

Reply:
yeah, that's a good idea to call it out in our docs somewhere. i think maybe the Internationalization section instead?